firedrake/plot.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import absolute_import
+import numpy as np
+from ufl import cell
+from firedrake import Function, SpatialCoordinate
+
+
+def plot_mult(functions, num_points=100):
+    try:
+        import matplotlib.pyplot as plt
+        from matplotlib.widgets import Slider
+    except ImportError:
+        raise RuntimeError("Matplotlib not importable, is it installed?")
+    figure, ax = plt.subplots()
+    func_axis = plt.axes([0.25, 0.025, 0.65, 0.03], axisbg='lightgoldenrodyellow')
+    func_slider = Slider(func_axis, "Func Select", 0.1, len(functions), valinit=0)
+    func_slider.valtext.set_text('0')
+
+    def _update(val):
```

Comment:
No need to make these internal functions "private" by prefixing them with an underscore.